components/atoms/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import type { PropsWithChildren } from 'react';
+import type Props from 'types/testType';
+
+const ErrorMessage = ({ children, testId }: PropsWithChildren<Props>): JSX.Element => (
```

Comment:
May we  can use `React.FC` insted of ``

Something like this
```
type Props = {
  title: string,
};
const Page: React.FC<Props> = ({ title, children }) => ...
```

Notice that children isn’t defined in Props. Instead, it is already defined in the FC type. This is nice if you know this fact, but it might be a bit confusing if you don’t.

Reply:
in this case it makes no difference.
With `React.FC` not possible pass generic property to component.
[example](https://github.com/DmitriyFirsov/react-next-rest-base/blob/master/components/molecules/Tabs/Tab.tsx#L8)